docs/reference/search/run-a-search.asciidoc

```diff
@@ -0,0 +1,285 @@
+[[run-a-search]]
+== Run a search
+
+You can use the <<search-search,search API>> to search data stored in
+one or more {es} indices.
+
+The API can runs two types of searches, depending on how you provide
+<<search-query,queries>>:
+
+<<run-uri-search,URI searches>>::
+  Queries are provided through a query parameter. URI searches tend to be
+  simpler and best suited for testing.
+
+<<run-request-body-search,Request body searches>>::
+  Queries are provided through the JSON body of the API request. These queries
+  are written in <<query-dsl,Query DSL>>. We recommend using request body
+  searches in most production use cases.
+
+[WARNING]
+====
+If you specify both a query in both the URI and request body, the search API
```

Comment:
Suggest removing the first "both" to read:
> If you specify a query in both the URI and request body...

Reply:
Good catch. Fixed with d05ea62df90.